plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/S3HudiQueryRunner.java

```diff
@@ -13,6 +13,7 @@
  */
 package io.trino.plugin.hudi;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
Still failing with below error in my laptop. Does the main method work in your environment? 
```
java.lang.reflect.InaccessibleObjectException: Unable to make field private final byte[] java.lang.String.value accessible: module java.base does not "opens java.lang" to unnamed module @378542de
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
	at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
	at org.apache.hudi.common.util.ObjectSizeCalculator$ClassSizeInfo.<init>(ObjectSizeCalculator.java:246)
	at org.apache.hudi.common.util.ObjectSizeCalculator.getClassSizeInfo(ObjectSizeCalculator.java:140)
	at org.apache.hudi.common.util.ObjectSizeCalculator.visit(ObjectSizeCalculator.java:158)
	at org.apache.hudi.common.util.ObjectSizeCalculator.calculateObjectSize(ObjectSizeCalculator.java:124)
	at org.apache.hudi.common.util.ObjectSizeCalculator.getObjectSize(ObjectSizeCalculator.java:74)
	at org.apache.hudi.common.util.DefaultSizeEstimator.sizeEstimate(DefaultSizeEstimator.java:30)
	at org.apache.hudi.common.util.queue.BoundedInMemoryQueue.adjustBufferSizeIfNeeded(BoundedInMemoryQueue.java:154)
	at org.apache.hudi.common.util.queue.BoundedInMemoryQueue.insertRecord(BoundedInMemoryQueue.java:191)
	at org.apache.hudi.common.util.queue.IteratorBasedQueueProducer.produce(IteratorBasedQueueProducer.java:46)
	at org.apache.hudi.common.util.queue.BoundedInMemoryExecutor.lambda$null$0(BoundedInMemoryExecutor.java:106)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

Reply:
Yes, it does. You may have to add the following VM arguments
```
-ea --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED
```
This will be fixed in the upcoming Hudi release.

Reply:
Please leave a code comment.